### PR TITLE
Add allowed_debuginfos for release

### PR DESCRIPTION
**Pull request by xiangge**
Add allowed_debuginfos for release
JIRA： PDC-1971

### CODE REVIEW

File: pdc/apps/release/serializers.py [hunk 1]
```diff
@@ -91,12 +92,16 @@ class ReleaseSerializer(StrictSerializerMixin, serializers.ModelSerializer):
                                           allow_null=True,
                                           default=None)
     allow_buildroot_push = serializers.BooleanField(default=False)
+    allowed_debuginfos = ChoiceSlugField(slug_field='name',
+                                         many=True, queryset=Service.objects.all(),
+                                         allow_null=True, required=False, default=[])
```
Comment (hluk) [inline pdc/apps/release/serializers.py]: Shouldn't this be `allow_null=False`?

Comment (xiangge) [reply pdc/apps/release/serializers.py]: Actually it's not , this value can be set as null ([]).

**Comment by hluk** [comment]
@larry-shi I suggest renaming `allowed_debuginfos` to `debuginfo_services`. Would it be OK or is there any reason for the name?

File: pdc/apps/release/serializers.py (hunk 1 above)
Comment (hluk) [reply pdc/apps/release/serializers.py]: `allow_null` just means that you can pass `None` in serializer ([docs](http://www.django-rest-framework.org/api-guide/fields/#allow_null)). Setting value to `null` is disallowed in model (`[]` is not `null`).

Would anything break if you remove it?

Comment (xiangge) [reply pdc/apps/release/serializers.py]: Aha, thanks for the explain, remove this, and just copy from the ReleaseGroupSerializer.releases, they are similar.

Moved.

**Comment by larry-shi** [comment]
> @larry-shi I suggest renaming allowed_debuginfos to debuginfo_services. Would it be OK or is there any reason for the name?
@hluk @xiangge 
Good idea, how about changing to allowed_debuginfo_services?

**Comment by hluk** [comment]
> @hluk @xiangge
> Good idea, how about changing to allowed_debuginfo_services?

@larry-shi In that case I would need to change `push_targets` to `allowed_push_targets` (in other PR). :) Probably a good idea.

**Comment by xiangge** [comment]
@larry-shi good idea

**Comment by xiangge** [comment]
Changed to allowed_debuginfo_services

**Comment by simozhan** [comment]
Just one thing, next time please commit another patch for your fix if someone already pull the codes and start review, -f push needs the reviewer to resolve conflicts. You can merge the commits together after you get approved by others and then -f push. Thanks :)

**Comment by xiangge** [comment]
@simozhan ok, good suggestions.

**Comment by xiangge** [comment]
@hluk , @simozhan and @larry-shi , pretty cool, thx guys!!

**Pull request merged**